.github/ISSUE_TEMPLATE/specification-issue.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Specification Issue
+about: Create a new issue related to the OpenAPI Specification
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**PLEASE NOTE:** This project only maintains the written [OpenAPI Specification](../blob/master/versions/3.0.2.md) and a few other supporting artifacts. No OpenAPI-related tooling lives here. If your issue is with a documentation format, UI, editor, programming framework, or other [OpenAPI implementation](../blob/master/IMPLEMENTATIONS.md), please post your issue in the appropriate project repository or contact the provider.
+
+_Please delete the above notice before submitting._
```

Comment:
As an alternative, the text above could be wrapped in an HTML comment, so would be visible during issue creation, but not when the submitted issue was viewed. Thus saving the submitter from having to delete the advice paragraph.

Reply:
@MikeRalphson, you mean a `<!-- comment like this -->`, right? 

The tradeoff is that we won't be able to create links that work in preview mode. The entire notice will be hidden in the preview and in the final issue description, only visible when editing. In that case, maybe I'd take the relative links out of the text, and add absolute links to the end so the user can copy and paste to get to the referenced pages.

> \<!--
> 
> PLEASE NOTE:  This project only maintains the written OpenAPI Specification and a few other supporting artifacts. No OpenAPI-related tooling lives here. If your issue is with an API documentation component, UI, editor, programming framework, or other OpenAPI implementation, please post your issue in the appropriate project repository or contact the provider.
> 
> LEARN MORE:
> * OpenAPI Specification
>    https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md
> * OpenAPI Implementations
>    https://github.com/OAI/OpenAPI-Specification/blob/master/IMPLEMENTATIONS.md
> 
> \-->

How does that look?

Reply:
Apart from the links to the current version of the specification, which would create a bit more work for the release manager to keep up to date, this LGTM. Maybe we could just link to the `versions` directory, though it's not ideal?